pkg/integrations/v4/logs/cfg_template.go

```diff
@@ -152,3 +152,29 @@ var fbLuaScriptFormat = `function {{ .FnName }}(tag, timestamp, record)
     -- If there is not any matching conditions discard everything
     return -1, 0, 0
  end`
+
+var tooManyFilesWarnMsg = `
+The amount of open files targeted by your Log Forwarding configuration
+files ({{ .TotalFiles }}) exceeds the recommended maximum (1024). The Operating System
+may kill the Log Forwarder process or not even allow it to start.
+These are the amount of files targeted by each of your configuration blocks:{{"\n"}}
+
+{{- range .LogsCfg }}
+{{- if .TargetFilesCnt }}
```

Comment:
I'd say to check for non-null `. File` here instead, just in case there is some file path that contains no files. In this case, we'd still want to print a "0" for that path.